src/theme/light.ts

```diff
@@ -151,4 +151,10 @@ export const lightTheme: ITheme = {
   },
 
   tooltip: inputTheme,
+
+  toaster: {
+    toastFg: '#111',
```

Comment:
Not sure if `toastFg` and `toastBg` is alright.
Would rather name it `fg` and `bg` respectively.

Reply:
I called it this way because this leaves room for 'containerBg' and 'containerFg' (which is supported by the toaster)

Reply:
I would like to leave it this way if you don't mind

Reply:
It's okay!